src/components/Hoverable/index.js

```diff
@@ -69,61 +68,46 @@ class Hoverable extends Component {
     }
 
     render() {
-        if (this.props.absolute && React.isValidElement(this.props.children)) {
-            return React.cloneElement(React.Children.only(this.props.children), {
-                ref: (el) => {
-                    this.wrapperView = el;
-
-                    // Call the original ref, if any
-                    const {ref} = this.props.children;
-                    if (_.isFunction(ref)) {
-                        ref(el);
-                    }
-                },
-                onMouseEnter: (el) => {
-                    this.setIsHovered(true);
-
-                    if (_.isFunction(this.props.children.props.onMouseEnter)) {
-                        this.props.children.props.onMouseEnter(el);
-                    }
-                },
-                onMouseLeave: (el) => {
-                    this.setIsHovered(false);
+        const child = _.isFunction(this.props.children) ? this.props.children(this.state.isHovered) : this.props.children;
 
-                    if (_.isFunction(this.props.children.props.onMouseLeave)) {
-                        this.props.children.props.onMouseLeave(el);
-                    }
-                },
-                onBlur: (el) => {
-                    if (!this.wrapperView.contains(el.relatedTarget)) {
-                        this.setIsHovered(false);
-                    }
-
-                    if (_.isFunction(this.props.children.props.onBlur)) {
-                        this.props.children.props.onBlur(el);
-                    }
-                },
-            });
+        if (!React.isValidElement(child)) {
```

Comment:
I am kind of worried this might cause a regression with something we didn't test, but I don't think we can avoid it, right?